### PR TITLE
feat: webpack version bump

### PR DESCRIPTION
**Pull request by kpsroka**

### CODE REVIEW

**Comment by kpsroka** [comment]
I managed to get the example app work with iOS, but not with Android. However, I want to focus on other stuff, so I'm pushing this to our working branch.

File: src/utils/makeReactNativeConfig.js [hunk 1]
```diff
@@ -126,56 +125,32 @@ const getDefaultConfig = ({
...
-                },
-              }),
-            ]
-          : []
-      ),
+            new webpack.SourceMapDevToolPlugin({
+              test: /\.(js|css|(js)?bundle)($|\?)/i,
+              filename: '[file].map',
+            }),
+            new webpack.optimize.ModuleConcatenationPlugin(),
```
Comment (thymikee) [inline src/utils/makeReactNativeConfig.js]: Use `optimization.concatenateModules`

File: src/utils/makeReactNativeConfig.js [hunk 2]
```diff
@@ -126,56 +125,32 @@ const getDefaultConfig = ({
...
-          : [
-              /**
+    ].concat(
+      dev
+        ? [
+            new webpack.HotModuleReplacementPlugin(),
+            new webpack.EvalSourceMapDevToolPlugin({
+              module: true,
+            }),
+            new webpack.NamedModulesPlugin(),
```
Comment (thymikee) [inline src/utils/makeReactNativeConfig.js]: Use `optimization.namedModules`

File: src/utils/makeReactNativeConfig.js [hunk 3]
```diff
@@ -126,56 +125,32 @@ const getDefaultConfig = ({
...
-        minify
-          ? [
-              new webpack.optimize.UglifyJsPlugin({
-                /**
-                 * By default, uglify only minifies *.js files
-                 * We need to use the plugin to configure *.bundle (Android, iOS - development) 
-                 * and *.jsbundle (iOS - production) to get minified. 
-                 * Also disable IE8 support as we don't need it.
-                 */
-                test: /\.(js|(js)?bundle)($|\?)/i,
```
Comment (thymikee) [inline src/utils/makeReactNativeConfig.js]: Curious if now webpack minimizes the files it processes

**Comment by thymikee** [comment]
@czystyl mind taking a look and see what can be done to run it on Android?

**Comment by czystyl** [comment]
I'm working on that.

**Pull request merged**